app/containers/Admin/ApplicationReview/ReviewSidebar.tsx

```diff
@@ -191,6 +195,27 @@ export const ReviewSidebar: React.FunctionComponent<Props> = ({
     </GenericConfirmationModal>
   );
 
+  const handleAddReviewComment = async ({commentText, isInternalComment}) => {
+    const commentType = isInternalComment ? 'internal' : 'general';
+    const {environment} = relay;
+    const variables = {
+      input: {
+        reviewComment: {
+          applicationReviewStepId: applicationReviewStep.rowId,
+          commentType: commentType.toUpperCase(),
+          description: commentText
+        }
+      }
+    };
+    await createReviewCommentMutation(
+      environment,
+      variables as createReviewCommentMutationVariables,
+      applicationReviewStep.id,
+      `ReviewSidebar_${commentType}Comments`
+    );
+    setShowAddCommentModal(false);
+  };
```

Comment:
I could not quite find a way to make this testable in the ReviewSidebar test (asserting that the mutation is called) as it's quite internal to the component... 🤔

Reply:
Do you mean that triggering `handleAddReviewComment` is the issue? 
The trick is to find it in the props of `AddReviewCommentModal` and call it directly. 
I did something similar here:
https://github.com/bcgov/cas-ciip-portal/blob/167365e41de8f02510a8873a7051d378ebf2f6e2/app/tests/unit/containers/Applications/ApplicationWizard.test.tsx#L210-L215

If you mean spying on the mutation I see that you've done that for another test already:

https://github.com/bcgov/cas-ciip-portal/blob/758dd771667548cb4f767496c093df5fcc268215/app/tests/unit/containers/Admin/ApplicationReview/ReviewSidebar.test.tsx#L220-L223

Reply:
> Do you mean that triggering handleAddReviewComment is the issue?

Yes, that is the issue because `handleAddReviewComment` is not a prop (and it shouldn't be) - so there's no access it to call it.

Reply:
Isn't it the value of the `onSubmit` prop of the `AddReviewCommentModal` below?

Reply:
Haha. Of course. It is. Tired eyes end of day when I wrote those tests yesterday!